build-service.py

```diff
@@ -179,6 +181,43 @@ def build_repo(repo_dir, branch, deploy_conf, service_conf):
     return image_name
 
 
+# create a new namespace specified in the deployment script and the namespace is not already in the system.
+# otherwise, if not specified, use default as namespace
+def create_namespace(kubernetes_api, deploy_conf):
+
+    if "namespace" not in deploy_conf["metadata"]:
+        # namespace is not specified in deplyoment config, just say the that namespace is default
+        deploy_conf["metadata"]["namespace"] = "default"
```

Comment:
Is / will this be documented in the template docs?

Reply:
I can add it after this PR gets approved